src/power_menu.rs

```diff
@@ -33,32 +33,53 @@ pub struct CliArgs {
     /// Show the menu items and exit
     #[arg(short, long)]
     pub list_items: bool,
+
+    /// Switch to elogind
+    #[arg(short, long, default_value = "false")]
+    pub elogind: bool,
+}
+
+// this should probably be somewhere else
```

Comment:
I have a pending refactoring to improve the project organization, but let's leave it here until then.

Reply:
OK